Source/common/SNTPostflightResult.h

```diff
@@ -0,0 +1,38 @@
+/// Copyright 2025 North Pole Security, Inc.
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+
+#import <Foundation/Foundation.h>
+
+#include "Source/common/SNTCommonEnums.h"
+
+@class SNTSyncState;
+
+@interface SNTPostflightResult : NSObject <NSSecureCoding>
+
+- (instancetype)initWithSyncState:(SNTSyncState *)syncState;
```

Comment:
I have mixed feelings about this. While being able to init directly from the sync state makes the code in SNTSyncPostflight much simpler, it requires importing SNTSyncState from santasyncservice which is crossing boundaries - nothing in common should should ever import from outside of common, nothing in each component folder should import from any other (only themselves or common).